src/data/gen2Banner-urls.ts

```diff
@@ -29,57 +57,196 @@ export const GEN2BANNER_URLS = [
   '/vue/build-a-backend/',
   '/react-native/build-a-backend/',
   '/javascript/build-a-backend/graphqlapi/',
+  '/angular/build-a-backend/graphqlapi/',
+  '/react/build-a-backend/graphqlapi/',
+  '/nextjs/build-a-backend/graphqlapi/',
+  '/vue/build-a-backend/graphqlapi/',
   '/javascript/build-a-backend/graphqlapi/connect-to-api/',
+  '/react/build-a-backend/graphqlapi/connect-to-api/',
+  '/angular/build-a-backend/graphqlapi/connect-to-api/',
+  '/vue/build-a-backend/graphqlapi/connect-to-api/',
+  '/nextjs/build-a-backend/graphqlapi/connect-to-api/',
   '/javascript/build-a-backend/graphqlapi/connect-existing-data/',
+  '/react/build-a-backend/graphqlapi/connect-existing-data/',
+  '/angular/build-a-backend/graphqlapi/connect-existing-data/',
+  '/vue/build-a-backend/graphqlapi/connect-existing-data/',
+  '/nextjs/build-a-backend/graphqlapi/connect-existing-data/',
   '/javascript/build-a-backend/graphqlapi/mutate-data/',
+  '/react/build-a-backend/graphqlapi/mutate-data/',
+  '/angular/build-a-backend/graphqlapi/mutate-data/',
+  '/vue/build-a-backend/graphqlapi/mutate-data/',
+  '/nextjs/build-a-backend/graphqlapi/mutate-data/',
   '/javascript/build-a-backend/graphqlapi/query-data/',
+  '/react/build-a-backend/graphqlapi/query-data/',
+  '/angular/build-a-backend/graphqlapi/query-data/',
+  '/vue/build-a-backend/graphqlapi/query-data/',
+  '/nextjs/build-a-backend/graphqlapi/query-data/',
   '/javascript/build-a-backend/graphqlapi/subscribe-data/',
+  '/react/build-a-backend/graphqlapi/subscribe-data/',
+  '/angular/build-a-backend/graphqlapi/subscribe-data/',
+  '/vue/build-a-backend/graphqlapi/subscribe-data/',
+  '/nextjs/build-a-backend/graphqlapi/subscribe-data/',
   '/javascript/tools/cli/graphqlapi/data-modeling/',
+  '/angular/tools/cli/graphqlapi/data-modeling/',
+  '/react/tools/cli/graphqlapi/data-modeling/',
+  '/nextjs/tools/cli/graphqlapi/data-modeling/',
+  '/vue/tools/cli/graphqlapi/data-modeling/',
   '/javascript/build-a-backend/graphqlapi/customize-authorization-rules/',
+  '/react/build-a-backend/graphqlapi/customize-authorization-rules/',
+  '/angular/build-a-backend/graphqlapi/customize-authorization-rules/',
+  '/vue/build-a-backend/graphqlapi/customize-authorization-rules/',
+  '/nextjs/build-a-backend/graphqlapi/customize-authorization-rules/',
   '/javascript/tools/cli/graphqlapi/custom-business-logic/',
+  '/angular/tools/cli/graphqlapi/custom-business-logic/',
+  '/react/tools/cli/graphqlapi/custom-business-logic/',
+  '/nextjs/tools/cli/graphqlapi/custom-business-logic/',
+  '/vue/tools/cli/graphqlapi/custom-business-logic/',
   '/javascript/tools/cli/graphqlapi/override-appsync/',
```

Comment:
There was a late move of this page from CLI - including JS all of these should be added to:
[platform]/build-a-backend/graphqlapi/modify-amplify-generated-resources/
Note this one also has a name change from /override-appsync/